src/igz_controls/components/validating-input-field/validating-input-field.component.js

```diff
@@ -199,11 +208,22 @@
                    ctrl.validationMaxLength;
         }
 
+        function isValueInvalid() {
+            if (angular.isDefined(ctrl.validationRules)) {
+                return lodash.some(ctrl.validationRules, ['isValid', false]);
+            }
+        }
```

Comment:
I feel like a `return false;` is missing here after the `if` statement.
You don't want this function to return `undefined`, right?
You want either `true` or `false`.
If the condition in the `if` statement does not pass, then you reach a flow where `undefined` is returned.

The question is, do you even need this `if`?

Considering:

```js
lodash.some(undefined, ['isValid', false])
// returns false

lodash.some(null, ['isValid', false])
// returns false

lodash.some([], ['isValid', false])
// returns false
```

I think you can just change the function body to:
```js
function isValueInvalid() {
    return lodash.some(ctrl.validationRules, ['isValid', false]);
}
```